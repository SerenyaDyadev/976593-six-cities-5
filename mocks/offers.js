const AVATAR_URL = `https://api.adorable.io/avatars/128`;

export default [
  {
    features: {
      type: `apartment`,
      badrooms: 2,
      adults: 4,
    },
    premium: false,
    price: 100,
    titie: `Wood and stone place`,
    rating: 3.5,
    things: [`Coffee machine`, `Dishwasher`, `Baby seat`, `Cabel TV`],
    pictures: [{
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }],
    description: `The concept, therefore, unbiased generates and provides a complex world.
    Liberation, by definition, transposes Taoism. Predicate calculus, as follows from the above,
    deduces hedonism, without taking into account the opinions of authorities.`,
    owner: {
      name: `John Snow`,
      super: false,
      picture: `${AVATAR_URL}/${Math.random()}`,
    }
  }, {
    features: {
      type: `house`,
      badrooms: 4,
      adults: 6,
    },
    premium: true,
    price: 10,
    titie: `Canal View Prinsengracht`,
    rating: 1.3,
    things: [`Heating`, `Kitchen`, `Fridge`, `Coffee machine`, `Dishwasher`, `Baby seat`, `Cabel TV`],
    pictures: [{
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }],
    description: `The concept, therefore, unbiased generates and provides a complex world.
    Liberation, by definition, transposes Taoism. Predicate calculus, as follows from the above,
    deduces hedonism, without taking into account the opinions of authorities.`,
    owner: {
      name: `Bob James`,
      super: true,
      picture: `${AVATAR_URL}/${Math.random()}`,
    }
  }, {
    features: {
      type: `hotel`,
      badrooms: 2,
      adults: 4,
    },
    premium: false,
    price: 200,
    rating: 4.8,
    titie: `Beautiful & luxurious studio at great location`,
    things: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Dishwasher`, `Baby seat`, `Cabel TV`],
    pictures: [{
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }],
    description: `The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.
    The world takes into account the gravitational paradox, although the opposite is accepted in officialdom.
    The axiom of syllogism contributes to ontological hedonism`,
    owner: {
      name: `Agent 007`,
      super: false,
      picture: `${AVATAR_URL}/${Math.random()}`,
    }
  }, {
    features: {
      type: `room`,
      badrooms: 1,
      adults: 2,
    },
    premium: true,
    price: 80,
    titie: `Canal View Prinsengracht`,
    rating: 2.9,
    things: [`Wi-Fi`, `Washing machine`, `Towels`, `Heating`, `Kitchen`, `Fridge`, `Coffee machine`, `Dishwasher`, `Baby seat`, `Cabel TV`],
    pictures: [{
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    },
    {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }, {
      src: `http://placeimg.com/260/200/arch`,
      description: `place description`,
    }],
    description: `The concept, therefore, unbiased generates and provides a complex world.
    Liberation, by definition, transposes Taoism. Predicate calculus, as follows from the above,
    deduces hedonism, without taking into account the opinions of authorities.`,
    owner: {
      name: `Many Pany`,
      super: true,
      picture: `${AVATAR_URL}/${Math.random()}`,
    }
  }
];
