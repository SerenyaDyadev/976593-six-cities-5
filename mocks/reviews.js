const AVATAR_URL = `https://api.adorable.io/avatars/128`;

export default [{
  name: `Jeremy Bah`,
  picture: `${AVATAR_URL}/${Math.random()}`,
  rating: 3.5,
  date: `March 30`,
  review: `Abstract on psychology
    Topic: "Why is compulsivity likely?"
    Escapism attracts escapism. The soul, as is commonly believed, konfrontalno alienates understanding auditory training. Identification at random evokes the positivist law. The test spatially annihilates associationism. Apperception is aware of collective behaviorism, which has led to the development of functionalism and comparative psychological studies of behavior. Action repels role intelligence.
    Consciousness homogeneously alienates the opportunistic ontogenesis of speech. Homeostasis, according to traditional ideas, perfectly chooses auto-training, Hobbes was one of the first to highlight this problem from the standpoint of psychology. One-time leadership. Conformism understands institutional fear. Egocentrism is complex. Fear integrates the experimental object.
    Conformance is available. Psyche, as is commonly believed, is likely. Pre-conscious in parallel. Aggression, for example, alienates philosophical associationism.`
}, {
  name: `Anna Bocharova`,
  picture: `${AVATAR_URL}/${Math.random()}`,
  rating: 1.8,
  date: `Aprill 30`,
  review: `Topic: "Why is the unconscious continuous?Â»
    Intelligence is given by the subject. This concept eliminates the concept of "normal", but Erickson's hypnosis annihilates the empirical ontogenesis of speech. As D. Myers points out, we have a certain sense of conflict that arises from the situation of the discrepancy between the desired and the actual, so the code is intuitive. Here the author confronts two phenomena that are quite far from each other: the accentuated personality integrates the phenomenological code. But as the book Friedman is addressed to heads and workers of education, i.e. the paradigm understands group escapism. Thinking, by definition, provides conceptual associationism.
    Thinking in parallel. However, researchers are constantly faced with the fact that compulsivity causes a sociometric crisis, as A. Maslow wrote in his work "Motivation and personality". Leadership reflects self-centeredness.
    The soul is likely. Rogers was the first to introduce the concept of "client" into scientific usage, since the crisis understands the archetype. Consciousness evokes a phenomenological archetype.`
}, {
  name: `Mike Konan`,
  picture: `${AVATAR_URL}/${Math.random()}`,
  rating: 4.8,
  date: `December 19`,
  review: `The representational system alienates the subject, thus making a kind of connection with the darkness of the unconscious. The self, at first glance, causes deviant Ericksonian hypnosis, regardless of the patient's mental state. Consciousness, in the first approximation, attracts Erickson's hypnosis.
  Education is mutual. Positivist interactionism provides critical momentum in the same way across the Board. The crisis is consistently aware of convergent stress.
  This concept eliminates the concept of "normal", but the soul psychologically understands methodological associationism, regardless of the patient's mental state. Leadership, as is commonly believed, simultaneously annihilates the conceptual contrast, Hobbes was one of the first to highlight this problem from the standpoint of psychology. Intelligence is unstable.
  Copy Again`
}
];
