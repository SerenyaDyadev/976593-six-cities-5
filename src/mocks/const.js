export const AVATAR_URL = `https://api.adorable.io/avatars/128`;

export const TITLES = [
  `Canal View Prinsengracht`,
  `Wood and stone place`,
  `Beautiful & luxurious studio at great location`,
  `Canal View Prinsengracht`,
];

export const PLACES = [
  `Apartment`,
  `House`,
  `Hotel`,
  `Private room`,
];

export const THINGS = [
  `Wi-Fi`,
  `Washing machine`,
  `Towels`,
  `Heating`,
  `Kitchen`,
  `Fridge`,
  `Coffee machine`,
  `Dishwasher`,
  `Baby seat`,
  `Cabel TV`
];

export const NAMES = [
  `John Snow`,
  `Many Pany`,
  `Iron Man`,
  `Spider Man`,
  `Agent 007`,
  `Bob James`,
  `Billi Willi`,
  `Anna Bocharova`
];

export const DESCRIPTION = [
  `The concept, therefore, unbiased generates and provides a complex world.
    Liberation, by definition, transposes Taoism. Predicate calculus, as follows from the above,
    deduces hedonism, without taking into account the opinions of authorities.`,

  `The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.
    The world takes into account the gravitational paradox, although the opposite is accepted in officialdom.
    The axiom of syllogism contributes to ontological hedonism`,

  `Education is mutual. Positivist interactionism provides critical momentum in the same way across the Board. The crisis is consistently aware of convergent stress.`,

  `This concept eliminates the concept of "normal", but the soul psychologically understands methodological associationism, regardless of the patient's mental state. Leadership, as is commonly believed, simultaneously annihilates the conceptual contrast, Hobbes was one of the first to highlight this problem from the standpoint of psychology. Intelligence is unstable.`,
];

export const COORDINATES = [
  {
    latitude: 52.3909553943508,
    longitude: 4.85309666406198
  }, {
    latitude: 52.369553943508,
    longitude: 4.85309666406198
  }, {
    latitude: 52.3909553943508,
    longitude: 4.929309666406198
  }, {
    latitude: 52.3809553943508,
    longitude: 4.939309666406198
  },
];

export const CITYES = {
  AMSTERDAM: `Амстердам`
};
