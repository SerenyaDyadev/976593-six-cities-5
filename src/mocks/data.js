export const cities = [`Paris`, `Cologne`, `Brussels`, `Amsterdam`, `Hamburg`, `Dusseldorf`];

export const offers = [
  {
    id: 0,
    isFavorite: false,
    isPremium: false,
    features: {
      type: `hotel`,
      badrooms: 1,
      adults: 2,
    },
    price: 100,
    title: `TITLE`,
    rating: 3.4,
    things: [`Wifi`, `Heating`, `Kitchen`, `Cable TV`, `Own bathroom`, `Hair-dryer`, `AC`],
    previewPictures: `img/apartment-small-01.jpg`,
    pictures: [`img/apartment-small-03.jpg`, `img/apartment-small-04.jpg`, `img/room-small.jpg`, `img/studio-photos.jpg`],
    description: `The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.`,
    owner: {
      id: 0,
      name: `Spider Man`,
      super: true,
      picture: `img/avatar-angelina.jpg`,
    },
    detailsMapZoom: 12,
    coordinates: [52.3909553943508, 4.85309666406198],
    mapZoom: 10,
    city: `Amsterdam`,
    cityCoordinates: [52.38333, 4.9],
  },
  {
    id: 2,
    isFavorite: true,
    isPremium: false,
    features: {
      type: `hotel`,
      badrooms: 2,
      adults: 4,
    },
    price: 200,
    title: `TITLE - 2`,
    rating: 4.5,
    things: [`Wifi`, `Heating`, `Kitchen`, `Cable TV`, `Own bathroom`, `Hair-dryer`, `AC`],
    previewPictures: `img/apartment-small-01.jpg`,
    pictures: [`img/apartment-small-03.jpg`, `img/apartment-small-04.jpg`, `img/room-small.jpg`, `img/studio-photos.jpg`],
    description: `The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.`,
    owner: {
      id: 1,
      name: `Iron Man`,
      super: false,
      picture: `img/avatar-angelina.jpg`,
    },
    detailsMapZoom: 12,
    coordinates: [52.3909553943508, 4.85309666406190],
    mapZoom: 10,
    city: `Amsterdam`,
    cityCoordinates: [52.38333, 4.9],
  },
  {
    id: 3,
    isFavorite: false,
    isPremium: true,
    features: {
      type: `private room`,
      badrooms: 1,
      adults: 6,
    },
    price: 300,
    title: `TITLE - 3`,
    rating: 4.9,
    things: [`Wifi`, `Heating`, `Kitchen`, `Cable TV`, `Own bathroom`, `Hair-dryer`, `AC`],
    previewPictures: `img/apartment-small-01.jpg`,
    pictures: [`img/apartment-small-03.jpg`, `img/apartment-small-04.jpg`, `img/room-small.jpg`, `img/studio-photos.jpg`],
    description: `The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.`,
    owner: {
      id: 3,
      name: `Billi Willi`,
      super: true,
      picture: `img/avatar-angelina.jpg`,
    },
    detailsMapZoom: 12,
    coordinates: [52.3909553943510, 4.85309666406201],
    mapZoom: 10,
    city: `Amsterdam`,
    cityCoordinates: [52.38333, 4.9],
  },
  {
    id: 4,
    isFavorite: true,
    isPremium: true,
    features: {
      type: `house`,
      badrooms: 6,
      adults: 3,
    },
    price: 400,
    title: `TITLE - 4`,
    rating: 2.1,
    things: [`Wifi`, `Heating`, `Kitchen`, `Cable TV`, `Own bathroom`, `Hair-dryer`, `AC`],
    previewPictures: `img/apartment-small-01.jpg`,
    pictures: [`img/apartment-small-03.jpg`, `img/apartment-small-04.jpg`, `img/room-small.jpg`, `img/studio-photos.jpg`],
    description: `The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.`,
    owner: {
      id: 4,
      name: `Agent 007`,
      super: true,
      picture: `img/avatar-angelina.jpg`,
    },
    detailsMapZoom: 12,
    coordinates: [52.3909553943515, 4.85309666406196],
    mapZoom: 10,
    city: `Amsterdam`,
    cityCoordinates: [52.38333, 4.9],
  },
];


export const reviews = [
  {
    id: 1,
    picture: `img/avatar-max.jpg`,
    name: `Spider Man`,
    rating: 2,
    review: `She carefully took a crumpled, half-torn ten-dollar bill from her purse and handed it to him. He shoved a white`,
    date: `2020-05-08T14:13:56.569Z`,
    super: true
  },
  {
    id: 2,
    picture: `img/avatar-max.jpg`,
    name: `Aqua Man`,
    rating: 5,
    review: `Be Happy! too! She carefully took a crumpled, half-torn ten-dollar bill from her purse and handed it to him. He shoved a white`,
    date: `2020-06-08T14:30:56.569Z`,
    super: true
  }
];
