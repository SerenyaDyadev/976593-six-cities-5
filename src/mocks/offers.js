export const offers = [
  {
    id: 0,
    isFavorite: false,
    isPremium: false,
    features: {
      type: `hotel`,
      badrooms: 1,
      adults: 2,
    },
    price: 100,
    title: `TITLE`,
    rating: 3,
    things: [`Wifi`, `Heating`, `Kitchen`, `Cable TV`, `Own bathroom`, `Hair-dryer`, `AC`],
    previewPictures: `img/apartment-small-01.jpg`,
    pictures: [`img/apartment-small-03.jpg`, `img/apartment-small-04.jpg`, `img/room-small.jpg`, `img/studio-photos.jpg`],
    description: `The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.`,
    owner: {
      id: 0,
      name: `Spider Man`,
      super: true,
      picture: `img/avatar-angelina.jpg`,
    },
    detailsMapZoom: 12,
    coordinates: [52.3909553943508, 4.85309666406198],
    ﾑ（tyZoom: 10,
    city: `Amsterdam`,
    cityCooridinates: [52.38333, 4.9],
  },
  {
    id: 2,
    isFavorite: true,
    isPremium: false,
    features: {
      type: `hotel`,
      badrooms: 2,
      adults: 4,
    },
    price: 200,
    title: `TITLE - 2`,
    rating: 3,
    things: [`Wifi`, `Heating`, `Kitchen`, `Cable TV`, `Own bathroom`, `Hair-dryer`, `AC`],
    previewPictures: `img/apartment-small-01.jpg`,
    pictures: [`img/apartment-small-03.jpg`, `img/apartment-small-04.jpg`, `img/room-small.jpg`, `img/studio-photos.jpg`],
    description: `The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.`,
    owner: {
      id: 1,
      name: `Iron Man`,
      super: false,
      picture: `img/avatar-angelina.jpg`,
    },
    detailsMapZoom: 12,
    coordinates: [52.3909553943508, 4.85309666406190],
    ﾑ（tyZoom: 10,
    city: `Amsterdam`,
    cityCooridinates: [52.38333, 4.9],
  },
  {
    id: 3,
    isFavorite: false,
    isPremium: true,
    features: {
      type: `private room`,
      badrooms: 1,
      adults: 6,
    },
    price: 300,
    title: `TITLE - 3`,
    rating: 5,
    things: [`Wifi`, `Heating`, `Kitchen`, `Cable TV`, `Own bathroom`, `Hair-dryer`, `AC`],
    previewPictures: `img/apartment-small-01.jpg`,
    pictures: [`img/apartment-small-03.jpg`, `img/apartment-small-04.jpg`, `img/room-small.jpg`, `img/studio-photos.jpg`],
    description: `The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.`,
    owner: {
      id: 3,
      name: `Billi Willi`,
      super: true,
      picture: `img/avatar-angelina.jpg`,
    },
    detailsMapZoom: 12,
    coordinates: [52.3909553943510, 4.85309666406201],
    ﾑ（tyZoom: 10,
    city: `Amsterdam`,
    cityCooridinates: [52.38333, 4.9],
  },
  {
    id: 4,
    isFavorite: true,
    isPremium: true,
    features: {
      type: `house`,
      badrooms: 6,
      adults: 3,
    },
    price: 400,
    title: `TITLE - 4`,
    rating: 4,
    things: [`Wifi`, `Heating`, `Kitchen`, `Cable TV`, `Own bathroom`, `Hair-dryer`, `AC`],
    previewPictures: `img/apartment-small-01.jpg`,
    pictures: [`img/apartment-small-03.jpg`, `img/apartment-small-04.jpg`, `img/room-small.jpg`, `img/studio-photos.jpg`],
    description: `The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.`,
    owner: {
      id: 4,
      name: `Agent 007`,
      super: true,
      picture: `img/avatar-angelina.jpg`,
    },
    detailsMapZoom: 12,
    coordinates: [52.3909553943515, 4.85309666406196],
    ﾑ（tyZoom: 10,
    city: `Amsterdam`,
    cityCooridinates: [52.38333, 4.9],
  },
];
