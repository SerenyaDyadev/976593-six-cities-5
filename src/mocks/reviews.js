export const reviews = [
  {
    id: 1,
    picture: `img/avatar-max.jpg`,
    name: `Spider Man`,
    rating: 2,
    review: `Be Happy!`,
    date: `2020-05-08T14:13:56.569Z`,
    super: true
  },
  {
    id: 2,
    picture: `img/avatar-max.jpg`,
    name: `Aqua Man`,
    rating: 5,
    review: `Be Happy! too`,
    date: `2020-06-08T14:30:56.569Z`,
    super: true
  }
];
