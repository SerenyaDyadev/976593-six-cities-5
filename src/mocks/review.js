import {NAMES, DESCRIPTION, AVATAR_URL} from "./const.js";
import {getRandomInteger, getElementArray} from "./utils.js";

const MONTHS = [
  `January`,
  `February`,
  `March`,
  `April`,
  `May`,
  `June`,
  `July`,
  `August`,
  `September`,
  `October`,
  `November`,
  `December`
];

export const generateReview = () => {

  return {
    name: getElementArray(NAMES),
    picture: `${AVATAR_URL}/${Math.random()}`,
    rating: getRandomInteger(0, 5),
    date: `${getElementArray(MONTHS)} ${getRandomInteger(1, 31)}`,
    review: getElementArray(DESCRIPTION)
  };
};
