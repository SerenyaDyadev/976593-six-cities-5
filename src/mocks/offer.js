import {CITYES, PLACES, TITLES, THINGS, NAMES, DESCRIPTION, AVATAR_URL, COORDINATES} from "./const.js";
import {getRandomInteger, getElementArray} from "./utils.js";
import {generateReview} from "./review";

const photoArrayLength = 6;

const getElenetsFromArray = (arr, begin = 0) => {

  return new Array(getRandomInteger(begin, arr.length - 1)).fill().map(() => getElementArray(arr));
};

const getPicturesArr = () => {

  return new Array(photoArrayLength).fill().map(() => {
    return {
      src: `http://picsum.photos/248/152?r=${Math.random()}`,
      description: `place description`,
    };
  });
};

export const generateOffer = () => {

  return {
    id: getRandomInteger(0, 30),
    isFavorite: Boolean(getRandomInteger(0, 1)),
    features: {
      type: getElementArray(PLACES),
      badrooms: getRandomInteger(1, 6),
      adults: getRandomInteger(1, 8),
    },
    isPremium: Boolean(getRandomInteger(0, 1)),
    price: getRandomInteger(0, 200),
    title: getElementArray(TITLES),
    rating: Number(`${getRandomInteger(0, 4)}.${getRandomInteger(0, 9)}`),
    things: getElenetsFromArray(THINGS, 3),
    pictures: getPicturesArr(),
    description: getElementArray(DESCRIPTION),
    owner: {
      name: getElementArray(NAMES),
      super: Boolean(getRandomInteger(0, 1)),
      picture: `${AVATAR_URL}/${Math.random()}`,
    },
    reviews: new Array(getRandomInteger(1, 6)).fill().map(generateReview),
    coordinates: getElementArray(COORDINATES),
    city: CITYES.AMSTERDAM
  };
};
