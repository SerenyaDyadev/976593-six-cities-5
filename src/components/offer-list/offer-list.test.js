import React from "react";
import renderer from "react-test-renderer";
import OfferList from "./offer-list";
import {offers} from "../../mocks/data";

jest.mock(`../offer-card/offer-card`, () => `OfferCard`);

it(`OfferList render`, () => {
  const tree = renderer
    .create(
        <OfferList
          offers={offers}
          classList={`cities__places-list tabs__content`}
          classCard={`place-card`}
          classImageWrapper={`cities__image-wrapper`}
          updateActiveOfferIdAction={()=>{}}
        />
    )

  .toJSON();

  expect(tree).toMatchSnapshot();
});
