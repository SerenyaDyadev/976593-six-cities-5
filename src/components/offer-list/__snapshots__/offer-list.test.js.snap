// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferList render 1`] = `
<div
  className="cities__places-list tabs__content places__list"
  onMouseOver={[Function]}
>
  <OfferCard
    classCard="place-card"
    classImageWrapper="cities__image-wrapper"
    offer={
      Object {
        "city": "Amsterdam",
        "cityCoordinates": Array [
          52.38333,
          4.9,
        ],
        "coordinates": Array [
          52.3909553943508,
          4.85309666406198,
        ],
        "description": "The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.",
        "detailsMapZoom": 12,
        "features": Object {
          "adults": 2,
          "badrooms": 1,
          "type": "hotel",
        },
        "id": 0,
        "isFavorite": false,
        "isPremium": false,
        "mapZoom": 10,
        "owner": Object {
          "id": 0,
          "name": "Spider Man",
          "picture": "img/avatar-angelina.jpg",
          "super": true,
        },
        "pictures": Array [
          "img/apartment-small-03.jpg",
          "img/apartment-small-04.jpg",
          "img/room-small.jpg",
          "img/studio-photos.jpg",
        ],
        "previewPictures": "img/apartment-small-01.jpg",
        "price": 100,
        "rating": 3,
        "things": Array [
          "Wifi",
          "Heating",
          "Kitchen",
          "Cable TV",
          "Own bathroom",
          "Hair-dryer",
          "AC",
        ],
        "title": "TITLE",
      }
    }
  />
  <OfferCard
    classCard="place-card"
    classImageWrapper="cities__image-wrapper"
    offer={
      Object {
        "city": "Amsterdam",
        "cityCoordinates": Array [
          52.38333,
          4.9,
        ],
        "coordinates": Array [
          52.3909553943508,
          4.8530966640619,
        ],
        "description": "The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.",
        "detailsMapZoom": 12,
        "features": Object {
          "adults": 4,
          "badrooms": 2,
          "type": "hotel",
        },
        "id": 2,
        "isFavorite": true,
        "isPremium": false,
        "mapZoom": 10,
        "owner": Object {
          "id": 1,
          "name": "Iron Man",
          "picture": "img/avatar-angelina.jpg",
          "super": false,
        },
        "pictures": Array [
          "img/apartment-small-03.jpg",
          "img/apartment-small-04.jpg",
          "img/room-small.jpg",
          "img/studio-photos.jpg",
        ],
        "previewPictures": "img/apartment-small-01.jpg",
        "price": 200,
        "rating": 3,
        "things": Array [
          "Wifi",
          "Heating",
          "Kitchen",
          "Cable TV",
          "Own bathroom",
          "Hair-dryer",
          "AC",
        ],
        "title": "TITLE - 2",
      }
    }
  />
  <OfferCard
    classCard="place-card"
    classImageWrapper="cities__image-wrapper"
    offer={
      Object {
        "city": "Amsterdam",
        "cityCoordinates": Array [
          52.38333,
          4.9,
        ],
        "coordinates": Array [
          52.390955394351,
          4.85309666406201,
        ],
        "description": "The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.",
        "detailsMapZoom": 12,
        "features": Object {
          "adults": 6,
          "badrooms": 1,
          "type": "private room",
        },
        "id": 3,
        "isFavorite": false,
        "isPremium": true,
        "mapZoom": 10,
        "owner": Object {
          "id": 3,
          "name": "Billi Willi",
          "picture": "img/avatar-angelina.jpg",
          "super": true,
        },
        "pictures": Array [
          "img/apartment-small-03.jpg",
          "img/apartment-small-04.jpg",
          "img/room-small.jpg",
          "img/studio-photos.jpg",
        ],
        "previewPictures": "img/apartment-small-01.jpg",
        "price": 300,
        "rating": 5,
        "things": Array [
          "Wifi",
          "Heating",
          "Kitchen",
          "Cable TV",
          "Own bathroom",
          "Hair-dryer",
          "AC",
        ],
        "title": "TITLE - 3",
      }
    }
  />
  <OfferCard
    classCard="place-card"
    classImageWrapper="cities__image-wrapper"
    offer={
      Object {
        "city": "Amsterdam",
        "cityCoordinates": Array [
          52.38333,
          4.9,
        ],
        "coordinates": Array [
          52.3909553943515,
          4.85309666406196,
        ],
        "description": "The delusion emphasizes intelligent common sense, although the opposite is accepted in the official language.",
        "detailsMapZoom": 12,
        "features": Object {
          "adults": 3,
          "badrooms": 6,
          "type": "house",
        },
        "id": 4,
        "isFavorite": true,
        "isPremium": true,
        "mapZoom": 10,
        "owner": Object {
          "id": 4,
          "name": "Agent 007",
          "picture": "img/avatar-angelina.jpg",
          "super": true,
        },
        "pictures": Array [
          "img/apartment-small-03.jpg",
          "img/apartment-small-04.jpg",
          "img/room-small.jpg",
          "img/studio-photos.jpg",
        ],
        "previewPictures": "img/apartment-small-01.jpg",
        "price": 400,
        "rating": 4,
        "things": Array [
          "Wifi",
          "Heating",
          "Kitchen",
          "Cable TV",
          "Own bathroom",
          "Hair-dryer",
          "AC",
        ],
        "title": "TITLE - 4",
      }
    }
  />
</div>
`;
